js/index.js

```diff
@@ -40,3 +40,69 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+
+const myH1 = document.querySelector(".cta-text h1")
+myH1.textContent = siteContent["cta"]["h1"];
+
+const myNav = document.querySelectorAll("nav a")
+myNav.forEach((nav, i)=>{
+  nav.textContent = siteContent["nav"]["nav-item-"+i]
+})
```

Comment:
We went through this together, but please let me know if you have any more questions on this. The main thing here is that `forEach`, as well as many other methods, pass more than each individual element in each invocation of the callback function: it also passes the index and even the whole array as arguments as well. Thus, you can add more parameters to your callback function to capture them if you need them in the code. 

In this case, we'd like the index, so we added an `i` as a parameter to the callback function. `forEach` will pass the index of the current item as the second argument, the value of which is captured with `i`.